reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7974,10 +7974,30 @@ public final void subscribe(Subscriber<? super T> actual) {
 	 * <p>
 	 * It is advised to return a {@link Publisher} derived from the original {@link Flux}
 	 * in all cases, as not doing so would keep the original {@link Publisher} open and
-	 * hanging with a single request and no cancellation. Suppressing early errors while
-	 * maintaining this link can be achieved with {@link #onErrorResume(Function)} and
-	 * {@link #empty()}.
-	 *
+	 * hanging with a single request by the terminal signal of the returned
```

Comment:
change whole end of paragraph to:
```
hanging with a single request until the inner {@link Publisher} terminates or the whole {@link Flux} is cancelled.
```
(please format as needed)

Reply:
fixed